### ISSUE TITLE
The 3-D design for the Space Bar should be implemented according to the spec laid out by the hi-fi mocks

### ISSUE THREAD

**Issue by kanyesthaker** [IrisHub/SpaceBar#128]

**Comment by kanyesthaker** [comment]
Blocked on #44

**Comment by kanyesthaker** [comment]
I suppose we no longer have a concept of "mocks" for the world. I think this will come back once we have proper version control, but this ticket is no longer relevant.
